app-frontend/src/app/pages/projects/edit/sharing/sharing.html

```diff
@@ -17,13 +18,10 @@ <h5 class="sidebar-title">Sharing</h5>
           {{policy.label}}
         </button>
       </div>
-    <i class="icon-info"></i>
-  </li>
-  <li class="separator">
-    <div class="label">
-      Share Page
     </div>
-    <div class="btn-group inline fixedwidth">
+    <div class="list-group-item sidebar-actions-group">
```

Comment:
Very minor change to the structure of your list-group-item's. Everything else is spot on. Good stuff.
```
<div class="list-group-item">
      <label>...</label>
      <div class="list-group-right btn-group">
        ...
      </div>
    </div>
```

Reply:
Great! looks good +1. You might want to coordinate with @Lknechtli on this pr and #2693